sources/packages/web/src/components/aest/students/ApplicationHeaderTitle.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <detail-header :headerMap="headerMap" />
+</template>
+
+<script lang="ts">
+import { defineComponent, onMounted, ref } from "vue";
+import { ApplicationSupplementalDataAPIOutDTO } from "@/services/http/dto";
+import { ApplicationService } from "@/services/ApplicationService";
+
+import DetailHeader from "@/components/generic/DetailHeader.vue";
+
+export default defineComponent({
+  components: { DetailHeader },
+  props: {
+    applicationId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props) {
+    const applicationData = ref<ApplicationSupplementalDataAPIOutDTO>();
+
+    const mapApplicationHeader = (
+      application: ApplicationSupplementalDataAPIOutDTO,
+    ): Record<string, string> => {
+      if (
+        application.applicationStartDate &&
+        application.applicationEndDate &&
+        application.applicationOfferingIntensity
+      ) {
+        return {
+          Name: application.studentFullName,
+          "Application #": application.applicationNumber,
+          Institution: application.applicationInstitutionName,
+          "Study dates": `${application.applicationStartDate} - ${application.applicationEndDate}`,
+          Type: application.applicationOfferingIntensity,
+        };
+      }
+      return {
+        Name: application.studentFullName,
+        "Application #": application.applicationNumber,
```

Comment:
According to AC, Application Number doesn't have `#`. 

![image](https://github.com/user-attachments/assets/c0a373e3-a700-4c50-82f7-e2d95f015bcb)

Reply:
The initial UI/UX had the "#" and it is common along the pages currently but the Ministry decided to drop it in favor of the "Application number". Please follow @dheepak-aot recommendation.

Reply:
updated